msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md

```diff
@@ -342,7 +342,10 @@ After you've previewed the manifest file in Visual Studio, you can now sync the
 :::image type="content" source="../assets/images/Tools-and-SDK-revamp/edit-manifest-for-visual-studio/vs-update-manifest1.png" alt-text="Update manifest in teams developer portal" lightbox="../assets/images/Tools-and-SDK-revamp/edit-manifest-for-visual-studio/vs-update-manifest1.png":::
 
 > [!NOTE]
-> The changes are updated to Teams Developer Portal. If you have some manual updates in Developer Portal, that can be overwritten. In the **Warning** dialog box you can select **Overwrite and update** or **Cancel**.
+>
```

Comment:
Add "The changes are updated to Teams Developer Portal" before to the note and the rest of the details should go in note.